src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -614,11 +614,7 @@ bool AsyncRPCOperation_sendmany::main_impl() {
             int wtxDepth = -1;
             {
                 LOCK2(cs_main, pwalletMain->cs_wallet);
-#if 0
-                const CWalletTx& wtx = pwalletMain->mapWallet[jso.hash];
-#else
-                const CWalletObjBase& wtx = *(pwalletMain->mapWallet[jso.hash]);
-#endif
+                const CWalletObjBase& wtx = *(pwalletMain->getMapWallet().at(jso.hash));
```

Comment:
the at() throws a std exception, the behaviour would be different from previous code